docs/2021/project-planning.md

```diff
@@ -0,0 +1,68 @@
+# Project Planning
+
+### Week 0 (08 March - 14 March)
+* Analyze GitHub API
+
+### Week 1 (15 March - 21 March)
+* Analyze GitHub Classroom
+* Sketch the UI page/representation interaction

```

Comment:
Perhaps add something along the lines of "Identity and document functional requirements".